samples/java_springboot/21.corebot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/FlightBookingRecognizer.java

```diff
@@ -48,8 +50,9 @@ public FlightBookingRecognizer(Configuration configuration, BotTelemetryClient t
             LuisRecognizerOptionsV3 recognizerOptions = new LuisRecognizerOptionsV3(luisApplication);
             recognizerOptions.setTelemetryClient(telemetryClient);
 
-            recognizer = new LuisRecognizer(recognizerOptions);
+            recognizer1 = new LuisRecognizer(recognizerOptions);
         }
+        this.recognizer = recognizer1;
```

Comment:
If these are the changes that we should apply just to have it as `final`, I'd prefer to rollback this